test/long-lines-long-reference-definitions.md

```diff
@@ -15,3 +16,9 @@
 [long-reference-definition-image]: https://example.com/long/long/long/long/long/long/long/long/long/long/long/long/long/image
 <!-- markdownlint-disable-next-line link-image-reference-definitions -->
 [long-reference-definition]: https://example.com/long/long/long/long/long/long/long/long/long/long/long/long/long
+
+<!-- markdownlint-configure-file {
```

Comment:
Adding this shouldn't be necessary? Also, I think this is the source of the invalid JSON test failure.